src/encoded/static/components/viz/Workflow/parsing-functions.js

```diff
@@ -219,7 +344,7 @@ export function parseAnalysisSteps(analysis_steps, parsingMethod = 'output'){
 
             var groupKeys = _.keys(groups);
             var filesNotInGroups = [];
-            var filesByGroup = _.reduce(stepInput.run_data.file, function(m, file, idx){
+            var filesByGroup = _.reduce(stepIOArgument.run_data.file, function(m, file, idx){
                 var incl = false;
```

Comment:
This entire section of code (grouping of files) works but is unwieldy; will likely be re-done completely at a future time but kept in non-refactored/cleaned-state for now.